aspnet/mvc/views/partial/sample/src/PartialViewsSample/Controllers/ArticlesController.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using Microsoft.AspNetCore.Mvc;
+using PartialViewsSample.ViewModels;
+
+namespace PartialViewsSample.Controllers
+{
+    public class ArticlesController : Controller
+    {
+        public IActionResult Index()
+        {
+            return View();
+        }
+
+        public IActionResult Read()
+        {
+            var viewModel = new Article()
```

Comment:
Sure. We're not showing that code anywhere that I know of, but it does clean it up a bit.